lib/fhir_client/client.rb

```diff
@@ -382,7 +382,7 @@ def get(path, headers)
 
     def post(path, resource, headers)
       url = URI(build_url(path)).to_s
-      FHIR.logger.info "POSTING: #{url}"
+      FHIR.logger.debug "POSTING: #{url}"
```

Comment:
Same here — and under `put`, `patch`, `delete`, and `head`, so I won't make the same comment over and over. 😄